app/src/main/java/com/hamdan/forzenbook/view/ForzenbookActivity.kt

```
@@ -97,4 +92,30 @@ class ForzenbookActivity : ComponentActivity() {
             }
         }
     }
+
+    override fun onDestroy() {
+        super.onDestroy()
+        if (!isChangingConfigurations) {
+            val fileDir = cacheDir.listFiles()
+            fileDir?.forEach {
+                deleteTempFiles(it)
+            }
+        }
+    }
+}
+
+private fun deleteTempFiles(file: File): Boolean {
```

Just include the part `file.delete()` lol why the directory logic? Let it fail if it's a directory because it shouldn't be?

---

thats fair, I was just seeing other ways of handling clean file deletion.